presto-client/package.json

```diff
@@ -6,5 +6,20 @@
   },
   "type": "commonjs",
   "main": "./src/index.js",
-  "typings": "./src/index.d.ts"
+  "typings": "./src/index.d.ts",
```

Comment:
Maybe worth review the order of these keys based on the order in https://docs.npmjs.com/cli/v10/configuring-npm/package-json 

You might use https://www.npmjs.com/package/sort-package-json